src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -3,8 +3,9 @@
 import java.util.Random;
 
 public class ColorSupplier {
+    private int randomNumberColor = new Random().nextInt(Color.values().length);
```

Comment:
In such case you will always have one randomNumberColor per object. 

You should create here only `Random` object.

`private final Random random = new Random();`